apps/explorer/src/ui/stories/ProgressBar.stories.tsx

```diff
@@ -11,11 +11,25 @@ export default {
 
 export const Default: StoryObj<ProgressBarProps> = {
     args: {
-        progress: 25,
+        progress: 75,
     },
     render: (args) => (
-        <div className="flex w-1/5">
+        <div className="flex w-1/2">
```

Comment:
can we change the main background color in Storybook for this component? right now it's white and so if you have `progress: 50` then half of the progress bar will be hidden which is kinda confusing

doesn't have to be a harsh gray like this but you get the point haha:

<img width="604" alt="image" src="https://user-images.githubusercontent.com/7453188/236567162-63fbbadd-8cf3-4551-876e-bc745e1092e1.png">